sdk/core/azure-core/src/environment_log_level_listener.cpp

```diff
@@ -138,11 +139,11 @@ EnvironmentLogLevelListener::GetLogListener()
         // insert unnecessary new lines.
         if (!message.empty() && message.back() == '\n')
         {
-          *os << std::flush;
+          std::cerr << std::flush;
         }
```

Comment:
I think this is no longer needed? Message will no longer ever be empty here, and it normally wouldn't end with `\n`.